.husky/pre-commit

```diff
@@ -2,3 +2,7 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npx lint-staged --relative
+
+nx prisma db format
+# Add the changes made by prisma format to the commit
+git add ./packages/db/src/schema.prisma
```

Comment:
We should use lint staged for this instead. That will automatically stage the changes and only run if the prisma file actually changed in the commit

Reply:
Done.